riva/client/asr.py

```diff
@@ -248,6 +248,12 @@ def print_streaming(
                 continue
             partial_transcript = ""
             for result in response.results:
+                if result.pipeline_states and len(result.pipeline_states.vad_probabilities) > 0:
+                    vad_prob_logs = "VAD States: "
+                    for vad_state in result.pipeline_states.vad_probabilities:
+                            vad_prob_logs += str(vad_state) + " "
+                    for i, f in enumerate(output_file):
```

Comment:
Are we not creating a new output file for dumping vad probabilities like in cpp client? Would that interfere with actual transcripts getting dumped into file? If both custom configuration is set to print vad probabilities and also print transcript flag is set to true, how does the content of output file look like?